### ISSUE TITLE
Not possible to move objects?

### ISSUE THREAD

**Issue by craftfortress** [gre/illuminated.js#17]
I can see the light position is accessible but if I change the objects parameters (instantiated with the light), their position on the canvas is not updated.  Am I doing something basically wrong or will the code need to be updated to handle dynamic movement of objects?

**Comment by gre** [comment]
I'm not sure if it's that but, do you recall `lighting.compute()` ?

**Comment by craftfortress** [comment]
Yep.
...
   lighting1.objects[1].topleft = new Vec2(0,0);
   lighting1.compute(canvas.width, canvas.height);

Dimension of object is not changed = (

**Comment by jasonmcleod** [comment]
@craftfortress you ever find a way to move objects? I'm fighting with it too :P

Thanks!

**Comment by jasonmcleod** [comment]
@craftfortress I'm back with good news! You have to adjust the points array with the newly calculated values, not just the topleft and bottomright vectors. 

I may work on a patch, but this worked for my needs.

**Comment by craftfortress** [comment]
Thanks! I was about to comment that I would look into when I have a free moment.  Cheers for the update.  Can I ask what you are working on? = )

**Comment by jasonmcleod** [comment]
Absolutely! I'm working on the 10,000th reboot of the multiplayer-dungeon-crawler/hack-and-slash that I never seen to finish :P
About every 4 months or so, I get the urge to work on (or more often, start over) on a game I played through high school called [Dransik](http://www.dransik.com/Screenshots.aspx). It's similar to the early Ultima games. It's really just a free roaming multiplayer RPG. I'm hoping to include some basic crafting in this version.. and hopefully _finish_ it :]

**Comment by gre** [comment]
sounds good, oh yeah topleft and bottomright can't be directly change. sounds like a bug, maybe we should rewrite Illuminated.js with more JavaScript ES6 style (setters, etc..).

good luck with your game.

**Comment by Zetaphor** [comment]
@jasonmcleod, @gre Can one of you please give an example as to what you mean?
